wagtail/wagtailusers/forms.py

```diff
@@ -54,68 +63,93 @@ def separate_username_field(self):
         return User.USERNAME_FIELD not in standard_fields
 
 
-class UserCreationForm(UsernameForm):
+class UserForm(UsernameForm):
     required_css_class = "required"
 
+    password_required = True
+
     error_messages = {
-        'duplicate_username': _("A user with that username already exists."),
+        'duplicate_username': _('A user with that username already exists.'),
```

Comment:
The reason is simply that I’m a single quote maniac. A maniac believing all quotes could be single, and all single quotes inside strings could be replaced with the character that was actually designed for this purpose: the apostrophe `’`. (The only exception being to quote a programming string in a string, and this would reasonable to escape them, like `'Here’s the code: \'Cool\''`)
But sorry about all that, I will revert these changes.